### PR TITLE
Add possibility to don't clip url in video description

### LINKED ISSUE #2218: Don't clip URLs in video descriptions.
**Is your enhancement request related to a problem? Please describe.**
image from `/watch?v=ZFUU59BAILc`
image
Can't copy a whole section, the links are clipped, ending with '...'

**Describe the solution you'd like**
Show all links in full. They may be clipped in HTML so they don't overflow (yet can still be copied).

### PR DESCRIPTION
**Pull request by MathiusD**
Closes #2218 

The purpose of this PR is to allow or not the clip of urls (as requested in [this issue](https://github.com/iv-org/invidious/issues/2218)).

As mentioned by @syeopite [here](https://github.com/iv-org/invidious/issues/2218#issuecomment-937724866), the goal was to make this modification configurable. Here it is configurable via the user preferences.

### CODE REVIEW

**Comment by unixfox** [comment]
IIRC the translation should be handled in weblate

**Comment by MathiusD** [comment]
> IIRC the translation should be handled in weblate

Indeed, I had put 2 translation keys because it seemed more appropriate, I can remove them if necessary

**Comment by SamantazFox** [comment]
I agree with matthew here.

There are already so many config options, and adding more for such simple details is simply going to be confusing for the average user. Imo, the description should stay "clean" (legible) for the majority of users, which is best achieved by keeping the URLs short.

I don't think copying entire descriptions is something one does often, and if that's needed, Imho a userscript is better suited at this purpose (i.e a JS snippet that would add a button to copy the description and some logic to clean out the links).

As an in-between solution, we could try to increase the size of the displayed text, while still clipping URLs that are too long.

**Comment by MathiusD** [comment]
In this sense, this pr is no longer necessary (unless you choose to lengthen the text of the links).
And it should be indicated in the original output that the user should turn to a user script

File: locales/en-US.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "preferences_autoplay_label": "Autoplay: ",
     "preferences_continue_label": "Play next by default: ",
     "preferences_continue_autoplay_label": "Autoplay next video: ",
+    "preferences_clip_urls_label":"Clip url displayed: ",
```
Comment (mhmdanas) [inline locales/en-US.json:66]:
```suggestion
    "preferences_clip_urls_label": "Clip url displayed: ",
```

File: locales/fr.json [hunk 2]
```diff
@@ -402,6 +402,7 @@
     "adminprefs_modified_source_code_url_label": "URL du dépôt du code source modifié",
     "footer_documentation": "Documentation",
     "footer_original_source_code": "Code source original",
+    "preferences_clip_urls_label":"Tronquer les liens affichés : ",
```
Comment (mhmdanas) [inline locales/fr.json:405]:
```suggestion
    "preferences_clip_urls_label": "Tronquer les liens affichés : ",
```

Comment (MathiusD) [reply locales/fr.json]: Resolved

File: locales/en-US.json (hunk 1 above)
Comment (MathiusD) [reply locales/en-US.json]: Resolved

**Pull request closed without merging**